test/session.test.ts

```diff
@@ -237,14 +236,14 @@ describe("Session", () => {
   });
 
   it("Should decode batched query operations", async () => {
-    const responses = await session.call([
+    const responses = (await session.call([
       operation.query(
         "select status.state.short from Task where status.state.short is NOT_STARTED limit 1"
       ),
       operation.query(
         "select status.state.short from Task where status.state.short is NOT_STARTED limit 1"
       ),
-    ]);
+    ])) as any[];
```

Comment:
Do `await session.call<QueryResponse>` instead to get the correct type of response.

We should probably look into a way to automatically infer that from the operation types included.